content/en/_index.md

```diff
@@ -293,14 +412,29 @@ sample) it is acceptable to either log or leave a comment explaining what the
 developer should do.
 
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
     // Follow the Google Java style guide and catch the most specific type of Exception, instead of a more general one.
```

Comment:
Out of Scope: references to another doc should probably be followed by the URL.

Reply:
BTW - I looked around [here](https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions), and didn't find anything relevant. (Maybe this is an _Effective Java_-sort of thing?)

Reply:
It might be an effective Java thing - I copied this from the Java samples style guide, so we probably have it wrong in there. I'd be okay with shortening it to "Always catch the most specific type of Exception, instead of a more general one." - I can also do that in a different PR if need be.

Reply:
I went ahead and did it here. 👍